tests/stress.rs

```diff
@@ -64,13 +64,12 @@ unsafe extern "C" fn stress_functions(context: &dc_context_t) {
         dc_simplify_simplify(simplify, html, strlen(html) as libc::c_int, 1i32, 0i32);
 
     // FIXME
-    // assert_eq!(
-    //     strcmp(plain, b"line1\nline2\x00" as *const u8 as *const libc::c_char),
-    //     0,
-    //     "{:?}",
-    //     std::ffi::CStr::from_ptr(plain),
-    // );
-    // free(plain as *mut libc::c_void);
+    assert_eq!(
+        CStr::from_ptr(plain as *const libc::c_char).to_str().unwrap(),
+        "line1\nline2",
+    );
+    free(plain as *mut libc::c_void);
```

Comment:
It's easier to see what's wrong if we compare like this:

![Screenshot from 2019-05-02 23-26-07](https://user-images.githubusercontent.com/308049/57108072-ff909000-6d31-11e9-854c-a8862efa34f3.png)

Now we can at least see that `plain` ends with two spaces. Further debugging needed in `dc_simplify_simplify()`.